terraform/state_bucket/bucket.tf

```
@@ -0,0 +1,51 @@
+resource "aws_s3_bucket" "bucket" {
+  bucket = "osquery-foundation-terraform-state"
```

```suggestion
  bucket = "osquery-terraform-state"
```

My gut sense is strike foundation